Tests/_InternalTestSupportTests/Misc.swift

```diff
@@ -0,0 +1,124 @@
+import _InternalTestSupport
+
+import XCTest
+
+class TestGetNumberOfMatches: XCTestCase {
```

Comment:
nit: no subclasses of `TestGetNumberOfMatches` are introduced, so this should be marked as `final`

```suggestion
final class TestGetNumberOfMatches: XCTestCase {
```